python/tank/platform/engine.py

```diff
@@ -192,6 +194,10 @@ def __init__(self, tk, context, engine_instance_name, env):
         qt_abstraction.QtCore = qt.QtCore
         qt_abstraction.QtGui = qt.QtGui
 
+        # load the fonts. this will work if there is a QApplication instance
+        # available.
+        self._ensure_core_fonts_loaded()
```

Comment:
ah, makes complete sense! However, public feels perhaps a step too far - but that's more a gut reaction than anything else! Maybe we could add some your comments to the `_ensure_core_fonts_loaded ` doc string?